.circleci/config.yml

```diff
@@ -400,10 +402,7 @@ workflows:
   version: 2
   build:
     jobs: &build_jobs
-      - manual-trigger-required-for-fork:
-          filters:
-            branches:
-              only: /^pull\/.*$/
```

Comment:
This needs to be re-executed when pushing to upstream otherwise it'll stay in failed state